Doc/library/dataclasses.rst

```diff
@@ -96,8 +96,24 @@ Module contents
      generated.  The generated repr string will have the class name and
      the name and repr of each field, in the order they are defined in
      the class.  Fields that are marked as being excluded from the repr
-     are not included.  For example:
-     ``InventoryItem(name='widget', unit_price=3.0, quantity_on_hand=10)``.
+     are not included.  For example::
+       >>> @dataclass(repr=False)
+       ... class InventoryItem:
+       ...     name: str = 'widget'
+       ...     unit_price: float = 3.0
+       ...     quantity_on_hand: int = field(default=10, repr=False)
+       ... 
+       >>> InventoryItem()  # doctest: +ELLIPSIS
+       <__main__.InventoryItem at 0x...>
+   
+       >>> @dataclass
+       ... class InventoryItem:
+       ...     name: str = 'widget'
+       ...     unit_price: float = 3.0
+       ...     quantity_on_hand: int = field(default=10, repr=False)
+       ... 
+       >>> InventoryItem()
+       InventoryItem(name='widget', unit_price=3.0)
```

Comment:
This one should be put in the section about the `repr` of `field` and not the `repr` of the `@dataclass` decorator.